pygmt/src/plot3d.py

```diff
@@ -163,6 +162,12 @@ def plot3d(
     {a}
     {c}
     {f}
+    incols : str or 1d array
+        Choose which columns are x, y, color, and size, respectively if
+        input is provided via ``data``. E.g. ``incols=[0, 1]`` or
+        ``incols='0,1'`` if the x values are stored in the first
+        column and y values in the second one. Note: zero-based
+        indexing is used.
```

Comment:
```suggestion
    {i}
```
You can simply use `{i}` here since @meghanrjones added `incols` to `COMMON_OPTIONS` in decorators.py via #1300.